### PR TITLE
Hibernate/Panache doc fixes

### PR DESCRIPTION
**Pull request by FroMage**
As noted in #17728 review

### CODE REVIEW

File: docs/src/main/asciidoc/hibernate-reactive-panache.adoc [hunk 1]
```diff
@@ -981,7 +981,7 @@ That's all there is to it: with Panache, Hibernate Reactive has never looked so
 
 Hibernate Reactive with Panache relies on compile-time bytecode enhancements to your entities.
 
-It attempts to identity archives with Panache entities (and consumers of Panache entities)
+It attempts to idenfity archives with Panache entities (and consumers of Panache entities)
```
Comment (geoand) [inline docs/src/main/asciidoc/hibernate-reactive-panache.adoc:984]:
```suggestion
It attempts to identify archives with Panache entities (and consumers of Panache entities)
```

Comment (FroMage) [reply docs/src/main/asciidoc/hibernate-reactive-panache.adoc]: Well that was dumb :)

Comment (geoand) [reply docs/src/main/asciidoc/hibernate-reactive-panache.adoc]: Happens to the best!

**Comment by geoand** [comment]
Mind squashing the commits?

**Comment by FroMage** [comment]
Done.

**Review by geoand — APPROVED** [review]

**Review by gsmet — APPROVED** [review]

**Pull request merged**